doc/memos/rdm1.md

```diff
@@ -0,0 +1,112 @@
+RIOT Maintainers Community (RIOT-MC)
+Developer Memo: 1
+Author(s): Hauke Petersen, more here
+Date: December 2016
+
+***
+RIOT Developer Memo Format, Publishing and Maintenance Process
+***
+
+# Abstract
+
+This memo contains a glossary with expressions commonly used in RIOT and in the
+context of RIOT. The aim of this document is not only to put some names to often
+used abbreviations, but also to explain their interpretation withing the RIOT
+community.
+
+
+# Status
+
+This document is a product of the community of RIOT maintainers, and aims to
+represent the consensus within this community.
+The content of this document is licensed with a Creative Commons CC-BY-SA
+license.
+
+# Terminology
+
+See below.
+
+
+# 1. Introduction
+
+As with many open source communities, there are a number of expressions, that
+are being commonly used among community members and community documents. Though
+most of these expressions are well known among developers, some of them are
+specific or are used with not-so-common semantic in the context of RIOT.
+
+This document provides a list of the most commonly used abbreviations and
+bash-words in RIOT. The scope of this document is not only to map these
+expressions to some more meaningful names, but also to explain their
+interpretation inside the RIOT community and to give some links to further
+information.
+
+Another aim of this RDM is further to act as a referencing target for future any
+RMDs.
+
+
+# 2. RIOT Glossary
+
+**ACK:** Acknowledgment of a pull request (PR) from a RIOT maintainer.
+
+**IoT:** Internet of Things - the thing everyone talks about theses days. In the
```

Comment:
I would rephrase. I guess these memos are supposed to survive the current hype. How about something like:
"The Internet of things has many different definitions dependent on the context. In the context of RIOT..."